integration/publisher_queue_test.go

```diff
@@ -44,6 +44,24 @@ func (suite *serviceBusQueueSuite) TestCreatePublisherUsingExistingQueue() {
 	}
 }
 
+// TestCreatePublisherWithDeadLetterForwardUsingNewQueue tests the creation of a publisher for a new queue
+func (suite *serviceBusQueueSuite) TestCreatePublisherWithDeadLetterForwardUsingNewQueue() {
+	suite.T().Parallel()
```

Comment:
cache the suite.T()
otherwise you might run require calls on whatever the current T is in the suite...